frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/premium.unit.spec.tsx

```diff
@@ -0,0 +1,169 @@
+import { screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createMockTokenFeatures } from "metabase-types/api/mocks";
+
+import { FONTS_MOCK_VALUES, getMockResource, setup } from "./setup";
+
+describe("Static Embed Setup phase - EE, with token", () => {
+  describe.each([
+    {
+      resourceType: "dashboard" as const,
+    },
+    {
+      resourceType: "question" as const,
+    },
+  ])("$resourceType", ({ resourceType }) => {
+    describe("Overview tab", () => {
+      it("should render content", () => {
+        setup({
+          props: {
+            resourceType,
+          },
+          activeTab: "Overview",
+          hasEnterprisePlugins: true,
+          tokenFeatures: createMockTokenFeatures({ whitelabel: true }),
+        });
+
+        expect(screen.getByText("Setting up a static embed")).toBeVisible();
+
+        const link = screen.getByRole("link", {
+          name: "documentation",
+        });
+        expect(link).toBeVisible();
+        expect(link).toHaveAttribute(
+          "href",
+          "https://www.metabase.com/docs/latest/embedding/static-embedding.html?utm_source=pro-self-hosted&utm_media=static-embed-settings-overview",
+        );
+      });
+
+      it("should support hiding metabase links feature", () => {
+        setup({
+          props: {
+            resourceType,
+          },
+          activeTab: "Overview",
+          hasEnterprisePlugins: true,
+          tokenFeatures: createMockTokenFeatures({ whitelabel: true }),
+          showMetabaseLinks: false,
```

Comment:
Can you add the case for `showMetabaseLinks: false` in the other 2 files `common.*` and `enterprise.*` to ensure the links are still visible in those cases?

I know the tests look a bit verbose, but to be fair, we don't have a ton of places that link to our doc. IIRC when I was working on the project there were only 16 locations I need to hide.